projects/commudle-admin/src/app/app.component.ts

```diff
@@ -1,54 +1,52 @@
-import { Component, Inject, PLATFORM_ID, OnInit } from '@angular/core';
-import { ApiRoutesService } from 'projects/shared-services/api-routes.service';
-import { environment } from '../environments/environment';
-import { LibAuthwatchService } from 'projects/shared-services/lib-authwatch.service';
-import { NbMenuItem, NbSidebarService, NbWindowService, NbWindowState, NbIconLibraries } from '@nebular/theme';
-import { faBars } from '@fortawesome/free-solid-svg-icons';
-import { ICurrentUser } from 'projects/shared-models/current_user.model';
-import { Router } from '@angular/router';
-import { DOCUMENT, isPlatformBrowser } from '@angular/common';
-import { Title } from '@angular/platform-browser';
-import { ActionCableConnectionSocket } from 'projects/shared-services/action-cable-connection.socket';
-import { UserPersonalDiscussionChatNotificationsChannel } from 'projects/shared-services/websockets/user-personal-discussion-chat-notifications.channel';
-import { CookieConsentComponent } from 'projects/shared-components/cookie-consent/cookie-consent.component';
-import { AppCentralNotificationService } from './services/app-central-notifications.service';
+import {CookieConsentService} from './services/cookie-consent.service';
+import {Component, Inject, OnInit, PLATFORM_ID} from '@angular/core';
+import {ApiRoutesService} from 'projects/shared-services/api-routes.service';
+import {environment} from '../environments/environment';
+import {LibAuthwatchService} from 'projects/shared-services/lib-authwatch.service';
+import {NbIconLibraries, NbMenuItem, NbSidebarService, NbWindowService, NbWindowState} from '@nebular/theme';
+import {faBars} from '@fortawesome/free-solid-svg-icons';
+import {ICurrentUser} from 'projects/shared-models/current_user.model';
+import {Router} from '@angular/router';
+import {DOCUMENT, isPlatformBrowser} from '@angular/common';
+import {Title} from '@angular/platform-browser';
+import {ActionCableConnectionSocket} from 'projects/shared-services/action-cable-connection.socket';
+import {AppCentralNotificationService} from './services/app-central-notifications.service';
+import {CookieConsentComponent} from '../../../shared-components/cookie-consent/cookie-consent.component';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent implements OnInit {
-  private isBrowser: boolean = isPlatformBrowser(this.platformId);
   sideBarNotifications = false;
   sideBarState = 'collapsed';
-
   faBars = faBars;
   currentUser: ICurrentUser;
   userContextMenu: NbMenuItem[] = [
-    { title: 'Logout', link: '/logout' },
+    {title: 'Logout', link: '/logout'},
   ];
+  cookieAccepted = false;
+  private isBrowser: boolean = isPlatformBrowser(this.platformId);
 
   constructor(
     @Inject(DOCUMENT) private document: Document,
-    @Inject(PLATFORM_ID) private platformId: Object,
+    @Inject(PLATFORM_ID) private platformId: object,
```

Comment:
Is this tested to work with object instead of Object

Reply:
Yes, it's working.

I did that because of [this](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).

![image](https://user-images.githubusercontent.com/60032753/107866676-3f8a1e00-6e99-11eb-8b7f-6dbc5c6563a3.png)

Reply:
Yeah, my linter also gave the same suggestion, but somehow I didn't pay that much attention :P